app/assets/stylesheets/comments.scss

```diff
@@ -22,15 +24,18 @@
     .media { margin: 5px; }
   }
 
+.comments > .comment {
+  padding: 10px 0px 10px 0px;
+  margin: 4px 10px 0px 10px;
+  border-bottom: 1px solid #eaeaea;
```

Comment:
Color literals like `#eaeaea` should only be used in variable declarations; they should be referred to via variable everywhere else.